Error building project using gradlew in Linux (Ubuntu - WSL context)

Hello,

Using the **master** branch of the repository, I've tried building the project in a WSL context (after of course installing the appropriate Android SDK and NDK versions).

By the way I had to set the NDK version on the **app/build.gradle** file, otherwise running gradlew command will automatically fetch the latest NDK version, the setup will look like this: `ndkVersion "21.4.7075529 `inside of the `android{...}` block

Unfortunately I get this error:
```bash
pxhoussem@DESKTOP-VQOQP66:~/testapp$ ./gradlew build

> Task :app:buildCMakeDebug[arm64-v8a] FAILED
C/C++: ninja: Entering directory `/home/pxhoussem/testapp/app/.cxx/Debug/4d1k475s/arm64-v8a'
C/C++: /home/pxhoussem/android/ndk/21.4.7075529/toolchains/llvm/prebuilt/linux-x86_64/bin/clang --target=aarch64-none-linux-android24 --gcc-toolchain=/home/pxhoussem/android/ndk/21.4.7075529/toolchains/llvm/prebuilt/linux-x86_64 --sysroot=/home/pxhoussem/android/ndk/21.4.7075529/toolchains/llvm/prebuilt/linux-x86_64/sysroot -DGRAPHICS_API_OPENGL_ES3 -DPLATFORM_ANDROID -D__ANDROID__ -I/home/pxhoussem/android/ndk/21.4.7075529/sources/android/native_app_glue -I/home/pxhoussem/testapp/app/src/main/cpp/deps/raylib -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security  -O0 -fno-limit-debug-info  -fPIC -std=c99 -MD -MT deps/raymob/CMakeFiles/raymoblib.dir/sensor.c.o -MF deps/raymob/CMakeFiles/raymoblib.dir/sensor.c.o.d -o deps/raymob/CMakeFiles/raymoblib.dir/sensor.c.o -c /home/pxhoussem/testapp/app/src/main/cpp/deps/raymob/sensor.c
C/C++: /home/pxhoussem/testapp/app/src/main/cpp/deps/raymob/sensor.c:72:54: error: no member named 'gyro' in 'struct ASensorEvent'
C/C++:                 State.inputs.gyroscopeAxis.x = event.gyro.x;
C/C++:                                                ~~~~~ ^
C/C++: /home/pxhoussem/testapp/app/src/main/cpp/deps/raymob/sensor.c:73:54: error: no member named 'gyro' in 'struct ASensorEvent'
C/C++:                 State.inputs.gyroscopeAxis.y = event.gyro.y;                                                                                                                                              C/C++:                                                ~~~~~ ^
C/C++: /home/pxhoussem/testapp/app/src/main/cpp/deps/raymob/sensor.c:74:54: error: no member named 'gyro' in 'struct ASensorEvent'
C/C++:                 State.inputs.gyroscopeAxis.z = event.gyro.z;
C/C++:                                                ~~~~~ ^
C/C++: 3 errors generated.

FAILURE: Build failed with an exception.
```

I've tried building using the [tag 5.5](https://github.com/Bigfoot71/raymob/tree/5.5) and it works fine. Although configuring the ndkVersion in the app/build.gradle is necessary also.

--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@ raymob is a simple implementation of [raylib](https://www.raylib.com/) for Andro
 
 ## Prerequisites
 
-**You will need SDK API 34 and NDK r21 (or higher, compatible with CMake 3.22.1).**
+**You will need SDK API 34 and NDK r23 (or higher, compatible with CMake 3.22.1).**
 
 If you already have this version of SDK and NDK without having Android Studio, you should still be able to compile the project using `gradlew.bat` for Windows or `gradlew` for Linux or MacOS.
 

